gap/base/recognition.gi

```diff
@@ -970,9 +970,37 @@ InstallGlobalFunction( "GetCompositionTreeNode",
 
 # Testing:
 
-RECOG.TestGroup := function(g,proj,size)
-  local l,r,ri,s,x,count,lvl,seedMT,seedRS,gens;
+RECOG.TestGroupOptions := rec(
+      # Number of times to test size of RecogniseGroup is right
```

Comment:
I had to read that several times to figure out what it was saying. Perhaps this is better?
```suggestion
      # Number of times to test whether the recognized size is right
```